src/index.html

```diff
@@ -4,9 +4,43 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="styles/index.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main class="main-container">
+      <section class="calendar">
```

Comment:
Add start day and month length modifiers for calendar
![image](https://user-images.githubusercontent.com/30314756/148017895-7dec5bfb-a7cb-42a6-ad7c-40cb79a3b887.png)